web-components/src/components/icons/CogIcon.tsx

```diff
@@ -13,13 +16,25 @@ export const CogIcon = ({ linearGradient, ...props }: Props) => (
   >
     <path
       d="M20.9664 10.0728L18.4842 9.6671C18.3388 9.16898 18.1402 8.69018 17.8915 8.237L19.3405 6.17186C19.4577 6.00512 19.4376 5.7779 19.2935 5.63342L17.6601 4.00088C17.5144 3.85556 17.2851 3.83666 17.1179 3.9572L15.0838 5.41922C14.6265 5.16638 14.1435 4.96562 13.6424 4.81988L13.2094 2.34776C13.1741 2.147 12.9998 2 12.7957 2H10.4857C10.2799 2 10.1043 2.1491 10.0711 2.35238L9.66962 4.81106C9.16562 4.95596 8.68178 5.15462 8.22608 5.4041L6.19748 3.9551C6.0299 3.8354 5.80184 3.85472 5.6561 3.99962L4.02356 5.63216C3.8795 5.77622 3.85934 6.00302 3.97652 6.16976L5.4041 8.21222C5.15 8.67212 4.94756 9.1589 4.80014 9.66542L2.35112 10.0732C2.14868 10.1068 2 10.2824 2 10.4874V12.7974C2 13.0011 2.14616 13.1754 2.3465 13.2111L4.79552 13.6453C4.9421 14.1506 5.14454 14.6374 5.39948 15.0985L3.95426 17.12C3.83498 17.2867 3.85388 17.5156 3.99878 17.6614L5.63174 19.2956C5.7758 19.4397 6.00302 19.4598 6.16976 19.3426L8.21516 17.91C8.67422 18.1624 9.15932 18.3628 9.66206 18.5081L10.072 20.9697C10.1052 21.1717 10.2803 21.32 10.4857 21.32H12.7957C12.9994 21.32 13.1737 21.1738 13.209 20.9735L13.6479 18.4997C14.1514 18.3514 14.634 18.1498 15.0885 17.897L17.1486 19.3422C17.3157 19.4602 17.5425 19.4397 17.687 19.2956L19.32 17.6614C19.4653 17.5156 19.4842 17.2859 19.3636 17.1187L17.8945 15.0788C18.1435 14.6252 18.3414 14.1456 18.4854 13.6474L20.971 13.2111C21.1722 13.1758 21.3183 13.0011 21.3183 12.7974V10.4874C21.3187 10.2816 21.1696 10.106 20.9664 10.0728Z"
-      stroke={linearGradient ? 'url(#linear0_cog_icon)' : 'currentColor'}
+      stroke={
+        linearGradient
+          ? 'url(#linear0_cog_icon)'
+          : props.disabled
+          ? '#8F8F8F'
```

Comment:
ditto let's not use hexcolor directly but rather colors from either the mui or tailwind theme

Reply:
same for all color values in this file

Reply:
This change sounds good. However maybe we should consider creating a task to update all SVGs to consistently use colors from the theme, not just this one? 

Also, I'm not sure how to use Tailwind colors in SVG attributes like `stopColor` or `stroke`. Do you have an example of how to do that?

One more thing: this color `#C5E6D1` is not part of our themes. It seems we use it only in SVGs. Do you think it would be a good idea to extend the theme to include it?

Reply:
ok let's leave it for now, I guess this will only become problematic once we want to ship more white-label apps that include those icons